islandora-deprecated/islandora_altmetrics
# Update to Islandora Badges

**JIRA Ticket**: (none)

* Other Relevant Links (Google Groups discussion, related pull requests, Release pull requests, etc.)
- [Roadmap Committee minutes, accepting Badges in Core](https://github.com/Islandora/islandora/wiki/Roadmap-Meeting-February-17th-2017-11:30-am-AST)
- [Islandora Badges as an Islandora component](https://groups.google.com/d/topic/islandora/mJMkDZlHGRI/discussion) 
- [Islandora Badges on GitHub](https://github.com/bondjimbond/islandora_badges)

# What does this Pull Request do?
As requested by @manez and the Roadmap Committee, this PR updates the Islandora Altmetrics module to become Islandora Badges (without changing the repository name).

# What's new?
Replaces the Altmetrics module with Badges and several submodules (including Altmetrics). For existing users of Altmetrics, Badges is now a required component and must also be enabled.

# How should this be tested?
- Update on a repo using the current version of Altmetrics
- Ensure that update doesn't break existing Altmetrics implementations, or if it does, document how
- Enable all submodules and configure blocks; then review objects with valid DOI, invalid DOI, and no DOI

# Interested parties
@manez @DiegoPino  @Islandora/7-x-1-x-committers
